server/controllers/UsersController.js

```diff
@@ -64,46 +67,96 @@ class UsersController {
   }
 
   /**
-    * @description - This method logs in user and return a token.
-    * @param {object} req - The request object bearing the email and password.
+    * @description - This method sends a login link to the user.
+    * @param {object} req - The request object bearing the email.
     * @param {object} res - The response object that is returned as json.
     * @returns {object} - The object with message.
     * @memberOf UserController
     * @static
     */
-  static userLogin(req, res) {
-    const { email, password } = req.body;
+  static userLoginStart(req, res) {
+    const { email } = req.body;
+    const lifeSpan = '1h';
+
     return User.findOne({
       where: {
         email
       }
     })
       .then((userFound) => {
-        if (!userFound || !bcrypt.compareSync(password, userFound.password)) {
-          return res.status(401).json({
+        if (!userFound) {
+          return res.status(404).json({
             errors: {
-              message: ['Invalid email or password']
+              message: ['User not found']
             }
           });
         }
-        if (bcrypt.compareSync(password, userFound.password)) {
-          const {
-            id,
-            fullName,
-            roleId
-          } = userFound;
-          const lifeSpan = 60 * 60 * 24;
-          return res.status(200).json({
-            status: 'success',
-            message: 'you are logged in',
-            user: {
-              fullName,
-              email,
-              roleId,
-              token: createToken(id, lifeSpan)
+
+        const userId = userFound.id;
+        const token = createToken(userId, lifeSpan);
+        const loginUrl = `${req.protocol}://${req.get('host')}`
```

Comment:
Do you not have the base url set in your env?

Reply:
I do not have the base URL in my `.env` file, would it be better if I moved it there?

Reply:
@mbilesanmi I do not have the base URL in my `.env` file, I will add it?